doc/release-notes/release-notes-bucash1.2.0.0.md

```diff
@@ -43,12 +43,16 @@ The old format continue to be supported and won't be deprecated, The old format
 that day the new format will be used as default new format automatically.
 A command line flag `-usecashaddr` and configuration parameter could be used to select which format to by default.
 
+Add limited support to BitPay Bitcoin Cash [addresses format](https://support.bitpay.com/hc/en-us/articles/115004671663-BitPay-s-Adopted-Conventions-for-Bitcoin-Cash-Addresses-URIs-and-Payme), this new format is accepted in any time an address is requested, and you can show the BitPay address form using the `showaddressforms` RPC. However BitPay-form addresses are not displayed in the GUI or as responses to any other RPCs.
+
```

Comment:
it's getaddressforms  not showaddressforms.

Reply:
thanks!